src/components/BackgroundImagePlaceholderDiv.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import cx from 'classnames';
+import { Icon } from '@mdi/react';
+import { mdiInformationOutline } from '@mdi/js';
+
+type Props = {
+  children?: any;
+  className?: string;
+  imageSrc: string;
+  hidePlaceholderOnHover?: boolean;
+};
+
+function BackgroundImagePlaceholderDiv(props: Props) {
+  const {
+    children, className,
+    imageSrc, hidePlaceholderOnHover,
+  } = props;
+
+  const [imageLoaded, setImageLoaded] = useState(false);
+
+  const backgroundImage = new Image();
```

Comment:
We should probably use a ref or memo to avoid creating a new Image object on every render....

Reply:
either memo this or React.memo whole component